front/src/modules/trainschedule/components/ManageTrainSchedule/ManageTrainScheduleMap/AddPathStepPopup.tsx

```diff
@@ -113,11 +99,11 @@ function RenderPopup({ pathProperties }: RenderPopupProps) {
     >
       <div className="details">
         <div className="details-track">
-          {featureInfoClick.feature.properties.extensions_sncf_track_name}
-          <small>{featureInfoClick.feature.properties.extensions_sncf_line_code}</small>
+          {featureInfoClick.feature.properties?.extensions_sncf_track_name}
+          <small>{featureInfoClick.feature.properties?.extensions_sncf_line_code}</small>
```

Comment:
You can remove the ? here
```suggestion
          {featureInfoClick.feature.properties.extensions_sncf_track_name}
          <small>{featureInfoClick.feature.properties.extensions_sncf_line_code}</small>
```

Reply:
Done